pkg/config/parse.go

```diff
@@ -0,0 +1,117 @@
+package config
+
+import (
+	"io/ioutil"
+	"runtime"
+
+	"github.com/BurntSushi/toml"
+	"github.com/kelseyhightower/envconfig"
+	validator "gopkg.in/go-playground/validator.v9"
+)
+
+// Config provides configuration values for all components
+type Config struct {
+	GoEnv                string         `validate:"required" envconfig:"GO_ENV"`
+	GoBinary             string         `validate:"required" envconfig:"GO_BINARY_PATH"`
+	LogLevel             string         `validate:"required" split_words:"true"`
```

Comment:
why do we need to split words on log level?

Reply:
The env variable is ATHENS_LOG_LEVEL, so that helps with the conversion to a camelcase property

Reply:
@rchakra3 why not just say `evnconfig: "ATHENS_LOG_LEVEL"` so that it's clear exactly what the corresponding ENV VAR is :)